docs/tfengine/schemas/iam_members.md

```diff
@@ -7,3 +7,7 @@
 | Property | Description | Type | Required | Default | Pattern |
 | -------- | ----------- | ---- | -------- | ------- | ------- |
 | iam_members | [Module](https://github.com/terraform-google-modules/terraform-google-iam) | object | false | - | - |
+| iam_members.^storage_bucket|project|organization|folder$ | - | array(object) | false | - | - |
```

Comment:
This file is not generated correctly. I believe it's because the pattern itself includes the `|` character which doesn't allow the table to be displayed correctly.

Reply:
Fixed.